com.unity.render-pipelines.universal/Shaders/Particles/ParticlesDepthNormalsPass.hlsl

```diff
@@ -0,0 +1,93 @@
+#ifndef UNIVERSAL_PARTICLES_LIT_DEPTH_NORMALS_PASS_INCLUDED
+#define UNIVERSAL_PARTICLES_LIT_DEPTH_NORMALS_PASS_INCLUDED
+
+#include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Core.hlsl"
+
+VaryingsDepthNormalsParticle DepthNormalsVertex(AttributesParticle input)
+{
+    VaryingsDepthNormalsParticle output = (VaryingsDepthNormalsParticle)0;
+    UNITY_SETUP_INSTANCE_ID(input);
+    UNITY_TRANSFER_INSTANCE_ID(input, output);
+    UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO(output);
+
+    VertexPositionInputs vertexInput = GetVertexPositionInputs(input.vertex.xyz);
+    VertexNormalInputs normalInput = GetVertexNormalInputs(input.normal, input.tangent);
+
+    half3 viewDirWS = GetWorldSpaceViewDir(vertexInput.positionWS);
+    #if !SHADER_HINT_NICE_QUALITY
+        viewDirWS = SafeNormalize(viewDirWS);
+    #endif
+
+    #ifdef _NORMALMAP
+        output.normalWS = half4(normalInput.normalWS, viewDirWS.x);
+        output.tangentWS = half4(normalInput.tangentWS, viewDirWS.y);
+        output.bitangentWS = half4(normalInput.bitangentWS, viewDirWS.z);
+    #else
+        output.normalWS = normalInput.normalWS;
+        output.viewDirWS = viewDirWS;
+    #endif
+
+    output.clipPos = vertexInput.positionCS;
+    output.color = GetParticleColor(input.color);
```

Comment:
same for color - only needed if alphatest is on.

Reply:
Fixed!